app/views/posts/index.html.erb

```diff
@@ -1,2 +1,8 @@
-<h1> Posts </h1>
-<p> Lists all posts for Joseph</p>
+<div class="post-container">
+  <%= render partial: 'shared/user-card', locals: { user: @user } %>
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <%= link_to post_id_path(@user, post), class: 'post-card-link' do %>
+      <%= render 'shared/post-card', post: post, id: post.id, brief: true, comment: true, all: false %>
+    <% end %>
+  <% end %>
```

Comment:
- [ ] You rendered the `post-card` partial here, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_comments` method you created in the post model to render the most recent 5 comments only here in `posts index` view.